go.mod

```diff
@@ -66,10 +66,60 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	k8s.io/klog v1.0.0 // indirect
 	k8s.io/kube-openapi v0.0.0-20221012153701-172d655c2280 // indirect
-	k8s.io/utils v0.0.0-20221101230645-61b03e2f6476 // indirect
+	k8s.io/utils v0.0.0-20221107191617-1a15be271d1d // indirect
 	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
 replace github.com/kubernetes-csi/external-snapshotter/client/v6 => ./client
+
+replace k8s.io/api => k8s.io/api v0.26.0-rc.0
```

Comment:
Do we need all these "replace" lines?  These were all removed by this PR: https://github.com/kubernetes-csi/external-snapshotter/pull/788/files